esp/src/eclwatch/DynamicESDLMethodWidget.js

```diff
@@ -48,9 +48,7 @@ define([
         idProperty: "__hpcc_id",
 
         init: function (params) {
-            if (this.inherited(arguments))
-                return;
-
```

Comment:
The Delay loader will spot and call "refresh" with the new params as needed, so do not remove the call to the base class, instead change:
```
         refresh: function () {
             this.refreshGrid();
         },
```
to:
```
         refresh: function (params) {
             this._params = params;
             this.refreshGrid();
         },
```
(You may need to also call the refreshState in there as well)